src/client/containers/layout.js

```diff
@@ -195,6 +195,11 @@ class Layout extends React.Component {
 						{' Revisions '}
 					</NavItem>
 				</Nav>
+				<Nav pullRight>
+					<NavItem href="/collections">
+						{' Collections '}
```

Comment:
I would like to see this menu item come *after* "Revisions" in the menu bar.
I believe in the code the order is reversed (because of the pullRight attributes), so in code it would need to be *before* Revisions